### PR TITLE
MTA 6.0 UI -- Bulk Analysis Configuration 

### PR DESCRIPTION
**Pull request by RichardHoch**
MTA 6.0

Resolves https://issues.redhat.com/browse/TACKLE-798

Preview: https://deploy-preview-566--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#assessing-and-analyzing-applications

### CODE REVIEW

**Review by sshveta — CHANGES_REQUESTED** [review]

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+. If you choose *Upload a local binary*, a window opens and you are prompted to *Upload a local bunary*. Either drag and drop a file into the area provided or click *Upload* and then select the file to upload.
+
+. Click *Next*.
+. Select one or more transformation targets:
+
+* Containerization
+* Quarkus
+* OpenJDK
+//OpenJDK 8 ?
```
[the remark this answers is not in the document]
Comment (RichardHoch) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:38]: Please check I have them all and in the right order.

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+. Click *Next*.
+. Select one of the following *Scope* options to better focus the analysis:
+
+* Application and internal dependencies only.
+* Application and all dependencies, including known Open Source libraries.
+* Select the list of packages to be analyzed manually. If you choose this option, type the file name and click *Add*.
+* Exclude packages. If you choose this option, type the name of the package name and click *Add*.
+
+. Click *Next*.
+. In *Adnaced*, set *Custom rules* by typing a name or by searching and clicking *Add Rules*.
```
Comment (kpunwatk) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:53]: Adnaced-->Advanced

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+====
+Analysis takes minutes to hours to run depending on the size of the application and the capacity and resources of the cluster.
+====
++
+[TIP]
+====
+{ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time.  With more cluster capacity, more analysis processes can be executed in parallel.
+====
+
+. When analysis is complete, you can click the *Report* link to see the resullts of the analysis.
```
Comment (kpunwatk) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:88]: resullts-->results

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+* Containerization
+* Quarkus
+* OracleJDK to OpenJDK
+** OpenJDK 11
+** OpenJDK 17
+* Linux
+* Camel
+* Azure AppService
+
```
Comment (kpunwatk) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:43]: The statement for Set Targets "**Select one or more target options in focus for the analysis**."
And also more targets are present now, Please refer the Set targets on latest MTA
Screenshot from 2022-11-03 20-56-41

Comment (RichardHoch) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:43]: @kpunwatk -- I need screen captures or a video -- what are the options in the application server migration to list? What are options when the user clicks Next?

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+* Exclude packages. If you choose this option, type the name of the package name and click *Add*.
+
+. Click *Next*.
+. In *Advanced*, set *Custom rules* by typing a name or by searching and clicking *Add Rules*.
+. Set any of the following options, if needed:
+
+* Target
+* Source(s)
+* Excluded rules tags: Rules with these tags are not processed. Add or delete as needed.
+* Enable transaction report: Select the checkbox to generate a DIVA report that displays the call stack, which executes operations on relational database tables. Transactions report that displays the call stack, which executes operations on relational database tables.
```
Comment (istein1) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:59]: Last sentence here seem to be redundant.

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
++
+[NOTE]
+====
+The Transactionד report is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them
```
Comment (istein1) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:63]: Remove the "ד"

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc (hunk 4 above)
Comment (kpunwatk) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:43]: Hi @RichardHoch, Thanks, below I've attached the video. If anything you need or have asked, please let me know.
[Screencast from 11-07-2022 12:17:15 PM.webm](https://user-images.githubusercontent.com/87791967/200243167-01c12d70-53b4-48d4-8e87-41f3ec5bbfb7.webm)

Comment (kpunwatk) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:43]: Right now there are 11 targets with three having sub-targets(with the drop-down options).

Comment (RichardHoch) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:43]: @kpunwatk -- Thanks. The video helped a lot!

**Comment by RichardHoch** [comment]
@kpunwatk -- please review again (Monday)

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+. Select the *Analysis mode* from the list. Valid options are:
+* Binary.
+* Source code.
+* Source code and dependencies.
+* Upload a local binary. This option only appears if you are analyzing a single aplication.
+
+. If you choose *Upload a local binary*, a window opens and you are prompted to *Upload a local binary*. Either drag and drop a file into the area provided or click *Upload* and then select the file to upload.
+
+. Click *Next*.
+. Select one or more transformation targets:
```
Comment (kpunwatk) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:33]: I think you forgot to change this line to **Select one or more target options in focus for the analysis**.

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc (hunk 4 above)
Comment (kpunwatk) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:51]: For Azure only "Azure App Service" is present

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc (hunk 7 above)
Comment (RichardHoch) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc]: @kpunwatk I actually do not want the command to be identical to the UI -- we don't usually write an entire instruction in bold, but I did change the command to fit the UI better.

Comment (kpunwatk) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc]: Oh, Got it, No Problem !
Thanks :+1:

Comment (RichardHoch) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc]: @kpunwatk So is the PR OK or is there anything else left?

**Review by sshveta — APPROVED** [review]

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+* Target
+* Source(s)
+* Excluded rules tags: Rules with these tags are not processed. Add or delete as needed.
+* Enable transaction report: Select the checkbox to generate a DIVA report that displays the call stack, which executes operations on relational database tables.
++
+[NOTE]
+====
+The Transactions report is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```
Comment (kpunwatk) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:72]: @RichardHoch I think by mistake there is long space in the statement "Red Hat does not recommend using them
in production" between line 71-72

**Review by kpunwatk — APPROVED** [review]
LGTM :+1:

**Comment by RichardHoch** [comment]
@sbeskin-redhat : Please review.

**Pull request merged**